doc/en/reference/reference.rst

```diff
@@ -1153,7 +1153,7 @@ specified plugins will be loaded.
 
 .. envvar:: PYTEST_PLUGINS
 
-Contains comma-separated list of modules that should be loaded as plugins:
+Contains comma-separated list of modules or entry points that should be loaded as plugins:
```

Comment:
With the https://github.com/pytest-dev/pytest/pull/12469 effort in mind, I think this should pick up some of the terminology as normalized there. Like third-party vs. local plugins.

This is currently talking about “modules” but they were never defined in https://docs.pytest.org/en/latest/how-to/writing_plugins.html, which introduces builtin/external/conftest types of plugins.

In this case, it might make sense to talk about importables (I suppose that's what “modules” means here, right?) and `pytest11` packaging metadata entry point names, if I understand correctly.

Reply:
Maybe.  But I believe it is out of scope of this PR, isn't it?

Reply:
It's in scope for as long as you refer to a thing that nobody understands what it is. You compare modules and entry points, but is it possible to understand from the updated sentence what the difference is? I know packaging in-depth, and it wasn't immediately clear to me. I imagine it'd be even more confusing to an average reader.

Reply:
Please either propose the wording you would like to see here, or I'll just revert this change.  I do not know how to write it differently to please you.

Reply:
I'll let @bluetech decide based on the discussion.

Reply:
how about something like this
```suggestion
Contains comma-separated list of :term:`importable modules <Module>`
or :ref:`entry point names <pip-installable plugins>` that should be
loaded as plugins:
```